packages-content-model/roosterjs-content-model-core/lib/corePlugin/ContentModelCopyPastePlugin.ts

```diff
@@ -250,6 +256,45 @@ class ContentModelCopyPastePlugin implements PluginWithState<CopyPastePluginStat
     };
 }
 
+/**
+ * @internal
+ * Exported only for unit testing
+ */
+export function adjustSelectionForCopyCut(pasteModel: ContentModelDocument) {
+    let selectionMarker: ContentModelSelectionMarker | undefined;
+    let firstBlock: ContentModelBlock | undefined;
+
+    iterateSelections(pasteModel, (_, tableContext, block, segments) => {
+        if (selectionMarker) {
+            if (
+                tableContext &&
+                block &&
+                firstBlock?.blockType == 'Paragraph' &&
+                firstBlock.segments.includes(selectionMarker) &&
+                block.blockType == 'Paragraph'
+            ) {
+                block.segments?.unshift(selectionMarker);
```

Comment:
I think we don't need to insert the selection marker into the table.

Reply:
You are right, removed this line.